AutomationCSharpTraining/AutomationCSharpTraining/PasswordPage.cs

```diff
@@ -12,21 +12,27 @@ class PasswordPage
     {
         IWebDriver _driver;
 
-        const string PasswordFieldLocator = "passwd";
-        const string EnterButtonLocator = "//button[@id=\"passp:sign-in\"]";
+        private static readonly By _passwordField = By.Name("passwd");
+        private static readonly By _enterButton = By.XPath("//button[contains(@id,\"sign-in\")]");
+        private static readonly string _commonPassword = "1wsx@QAZ";
+
         public IWebElement PasswordField { get; set; }
```

Comment:
Initialize your IWebElement properties straight here.
For example, public IWebElement PasswordField => _driver.FindElement(By.Id(By.Name(PasswordFieldLocator))). It will allow you to work with web elements at this class level.